CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ FEATURES:
   * Create genesis transactions with `gaiad init gen-tx` 
   * New genesis account keys are automatically added to the client keybase (introduce `--client-home` flag)
   * Initialize with genesis txs using `--gen-txs` flag
+* Context now has access to the application-configured logger
```

Comment:
Move to the new section up top above 0.15.1 as we've already released

Reply:
Moved.